lnrpc/rpc.proto

```diff
@@ -780,7 +780,14 @@ message SendRequest {
     any channel may be used.
     */
     uint64 outgoing_chan_id = 9;
+
+    /** 
+    An optional maximum total time lock for the route. If zero, there is no
+    maximum enforced.
+    */
+    uint32 cltv_limit = 10;
```

Comment:
I didn't add it into `QueryRoutes` just yet, because I'd first like to get rid of the k-shortest algorithm there. Otherwise I need to spend time now to work the cltv limit into the partial paths of k-shortest, while in the end it will be removed completely.

https://github.com/lightningnetwork/lnd/pull/2497